cobbler/validate.py

```diff
@@ -308,7 +308,7 @@ def validate_repos(repos: list, api, bypass_check: bool = False):
     return repos
 
 
-def validate_virt_file_size(num: Union[str, int, float]):
+def validate_virt_file_size(num: Union[str, int, float]) -> Union[str, float]:
```

Comment:
Something that I just thought of (not for this backport): I think it makes sense to create a custom type for `<<inherit>>`. `Union[str, float]` could then be `Union[inherit, float]`, which would be a lot clearer. Just an idea for the future, if you like it I can open a PR for main at some point.

Reply:
Let's create an issue for that. We talked about this before already but I never had the time to follow up on it in detail. I thought it was the Sentinel Object Pattern, right?

Reply:
Issue created: #3074